### PR TITLE
Create microprofile-config.travis.yml

### PR DESCRIPTION
**Pull request by nagesh4193**

### CODE REVIEW

**Comment by gururajrkatti** [comment]
Do we have build_script available for this ?

**Comment by nagesh4193** [comment]
I have added build script for microprofile-config and the success message in my local vm as below:

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] MicroProfile Config ................................ SUCCESS [ 13.330 s]
[INFO] MicroProfile Config Specification .................. SUCCESS [ 37.136 s]
[INFO] MicroProfile Config API ............................ SUCCESS [ 16.860 s]
[INFO] MicroProfile Config TCK ............................ SUCCESS [ 13.852 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:23 min
[INFO] Finished at: 2021-04-08T13:34:20+00:00
[INFO] Final Memory: 125M/450M
[INFO] ------------------------------------------------------------------------

File: m/microprofile-config/microprofile-config_ubuntu_16.04.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# Install dependencies
+sudo apt-get update -y
+sudo apt-get install -y openjdk-8-jdk maven git  
+
+export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-ppc64el
+export JRE_HOME=${JAVA_HOME}/jre
+export PATH=${JAVA_HOME}/bin:$PATH
+
+git clone https://github.com/eclipse/microprofile-config
```
Comment (gururajrkatti) [inline m/microprofile-config/microprofile-config_ubuntu_16.04.sh:27]: Pl. have the version number passed as an argument along with the default one used in the script as mentioned earlier. Also upload the LICENSE file

**Review by gururajrkatti — CHANGES_REQUESTED** [review]

**Comment by nagesh4193** [comment]
This is failing when version is added in the build specification, so I am closing this PR and would raise a new PR after the fix.

**Pull request closed without merging**